ledger/ledger-on-memory/src/main/scala/com/daml/ledger/on/memory/InMemoryLedgerReader.scala

```diff
@@ -41,3 +42,17 @@ class InMemoryLedgerReader(
 
   override def currentHealth(): HealthStatus = Healthy
 }
+
+object InMemoryLedgerReader {
+  private val StartIndex: Index = 0
+
+  def dispatcher: ResourceOwner[Dispatcher[Index]] =
+    ResourceOwner.forCloseable(
+      () =>
+        Dispatcher(
+          "in-memory-key-value-participant-state",
+          zeroIndex = StartIndex,
+          headAtInitialization = StartIndex,
+      ))
```

Comment:
These don't truly belong to either the reader or the writer. I suggest moving them to the package object.

Reply:
Done.